src/os_handlers.eliomi

```diff
@@ -10,6 +10,8 @@ val sign_up_handler : unit -> string -> unit Lwt.t
 val activation_handler :
   string -> unit -> Eliom_registration.Action.result Lwt.t
 
+val add_mail_handler : int64 -> unit -> string -> unit Lwt.t
```

Comment:
I changed it a little bit, I was checking twice if the user was connected.

`val add_mail_handler : unit -> string -> unit Lwt.t`

It looks better.